op-guide/requirement.md

```diff
@@ -0,0 +1,77 @@
+- The support for the Linux operating systems above include the deployment and operation in physical servers as well as in the mainstream virtualized environments like VMWare, KVM and XEM.
+
+## Server Requirements
+
+You can deploy and operate TiDB on the 64-bit universal hardware server platform in the Intel x86-64 architecture. The requirements and recommendations about server hardware configuration for development, testing and production environments are as follows:
+
+### Development and Testing Environments
+
+| Component | CPU     | Memory | Local Storage  | Network  | Instance Number (lowest requirement) |
+| :------: | :-----: | :-----: | :----------: | :------: | :----------------: |
+| TiDB    | 16 core+   | 16 GB+  | SAS, 200 GB+ | Gigabit network card | 1       |
+| PD      | 16 core+   | 16 GB+  | SAS, 200 GB+ | Gigabit network card | -       |
+| TiKV    | 16 core+   | 32 GB+  | SAS, 200 GB+ | Gigabit network card | 3       |
+|         |         |         |              | Total Server Number |  4      |     
+
+**Note**
+
+- Verify that in the test environment the TiDB and PD can be deployed on the same server.
+- In performance-related testing, do not use low-performance storage and network hardware configuration, in order to guarantee the correctness of the test result.
+
+### Production Environment
+
+| Component | CPU | Memory | Hard Disk Type | Hard Disk Number | Single Hard Disk Size | Network | Instance Number (lowest requirement) |
+| :-----: | :------: | :------: | :------: | :------: | ------ | :------: | :-----: |
+|  TiDB  | 32 core+ | 128 GB+ | SSD | minimum 2 | 500 GB+ | 2+ 10 Gigabit network card | 2 |
+| PD | 16 core+ | 32 GB+ | SSD | minimum 2 | 200 GB+ | 2+ 10 Gigabit network card | 3 |
+| TiKV | 32 core+ | 128 GB+ | SSD | minimum 2 | 200~500 GB. It is recommended to keep the size of TiKV hard disk within 500G in case it takes too long to restore data when the hard disk is damaged.  | 2+ 10 Gigabit network card | 3 | 
+| Monitor | 16 core+ | 32 GB+ | SAS | minimum 4 | 200 GB+ | 2+ 10 Gigabit network card | 1 |
+|     |     |     |      |      |      |  Total Server Number   |    9   |
+
+**Note**:   
+
+- In the production environment, you can deploy and operate TiDB and PD on the same server. If you have a higher requirement for performance and reliability, try to deploy them separately.
+- It is strongly recommended to use higher configuration in the production environment.
+
+## Network Requirements
+
+As an open source distributed NewSQL database, TiDB requires the following network port configuration to operate. Based on the TiDB deployment in actual environments, the administrator can enable relevant ports in the network side and host side. 
+
+| Component | Default Port | Description |
+| :--:| :--: | :-- |
+| TiDB |  4000  | access communication port of application and DBA tools|
```

Comment:
access communication port of application and DBA tools ->
the communication port for application and DBA tools